EssentialFeed/EssentialFeed/Image Comments/API/ImageCommentEndpoint.swift

```diff
@@ -0,0 +1,16 @@
+//
+// Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+
+public enum ImageCommentEndpoint {
+	case get
```

Comment:
Consider using associated values for the parameter necessary for this `get`:
```suggestion
	case get(UUID)
```